modules/virt-installing-qemu-guest-agent-on-windows-vm.adoc

```diff
@@ -13,7 +13,9 @@ For Windows virtual machines (VMs), the QEMU guest agent is included in the Virt
 ====
 To create snapshots of an online (Running state) VM with the highest integrity, install the QEMU guest agent.
 
-The QEMU guest agent takes a consistent snapshot by attempting to quiesce the VM file system as much as possible, depending on the system workload. This ensures that in-flight I/O is written to the disk before the snapshot is taken. If the guest agent is not present, quiescing is not possible and a best-effort snapshot is taken. The conditions under which the snapshot was taken are reflected in the snapshot indications that are displayed in the web console or CLI.
+The QEMU guest agent takes a consistent snapshot by attempting to quiesce the VM file system as much as possible, depending on the system workload. This ensures that in-flight I/O is written to the disk before the snapshot is taken. If the guest agent is not present, quiescing is not possible and a best-effort snapshot is taken. 
```

Comment:
I think its worth adding here about how in windows the freeze is being done by the VSS which also needs to be ready and responsive when the snapshot is taken.